grails-app/domain/org/broadinstitute/orsp/Issue.groovy

```diff
@@ -195,22 +195,22 @@ class Issue implements LogicalDelete<Issue> {
         properties
     }
 
-    transient LinkedHashMap<String, Object> getExtraPropertiesMap() {
-        LinkedHashMap<String, Object> extraproperties = new LinkedHashMap<>()
-        
-        Collection<String> collaborators = new ArrayList<String>()
-        getExtraProperties().collect {
-            if ( it.name != IssueExtraProperty.COLLABORATOR ) {
-                extraproperties.put(it.name, it.value)
-            } else {
-              collaborators.add(it.value);
-            }
+    /**
+     * Returns a map of IssueExtraProperty values for an issue, grouped by name.
+     *
+     * @return Map of extra properties, grouped by name
+     */
+    @SuppressWarnings("GroovyAssignabilityCheck")
```

Comment:
This is needed to stop the IDE from complaining about `propsByName.collectEntries`. It should know that `key` is a string and `list` is a `List<String>`, but doesn't.